src/ARMarker.js

```diff
@@ -70,13 +84,17 @@ class ARMarker extends HTMLElement {
                 let ctx = self.textureCanvas.getContext('2d');
                 ctx.clearRect(0, 0, self.textureCanvas.width, self.textureCanvas.height);
                 ctx.drawImage(self.image, 0, 0, self.textureCanvas.width, self.textureCanvas.height);
-        
+
             } else if(ev.data.marker.id == markerId) {
+                if(self.video.ended != 0 && self.contentProps.loop == 0)
```

Comment:
The problem with this line is when the video ended if I have to update the page to play again. When the marker goes out of the camera vision and then returns, the video doesn't play again. This is the expected behavior?